README.md

```diff
@@ -25,21 +25,21 @@ npm install enceeper-jslib --save
 
 ### Important notes
 
-The library assumes that the machine we are running is not hostile. If another hostile process can allocate a memory previously used by our app it can gain access to important key information. We do not zero variables after usage and also JS allocates memory dynamically, so by itself this would be a very hard task.
+The library assumes that the machine we are running is not hostile. If another hostile process can acquire a memory segment previously used by our app it could gain access to important key information. Given that JS allocates memory dymanically, even considering zeroing variables after usage by itself would be a very hard task.
```

Comment:
I like the first part, but not the ending: "by itself would be a very hard task". Do you think this is better: "could add unnecessary complexity without accomplishing its goal".

Reply:
good one.!